resolwe/process/fields.py

```diff
@@ -512,7 +515,13 @@ def __init__(self, data_id, field, cache):
 
             field_type = field_descriptor["type"].rstrip(":")
 
-            if field_type.startswith("list:"):
+            if field_type.startswith("list:data"):
+                # The only case where output field is of type
```

Comment:
Why is that?

Reply:
This is just how we currently do it. There are no processes in resolwe/resolwe-bio that would have data objects as outputs (except for workflows).